tools/reviewer/web/src/app/dashboard/reviews-panel/review/review-titlebar/review-titlebar.component.ts

```diff
@@ -1,18 +1,42 @@
-import { Diff } from '@/shared';
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+import { ReviewService } from '../review.service';
+
+import {
+  AuthService
+} from '@/shared/services';
+import { Diff } from '@/shared/shell';
+import { statusList } from './status-ui';
 
 @Component({
   selector: 'review-titlebar',
   templateUrl: './review-titlebar.component.html',
   styleUrls: ['./review-titlebar.component.scss']
 })
-export class ReviewTitlebarComponent {
-  @Input() diff: Diff.AsObject;
+export class ReviewTitlebarComponent implements OnInit {
+  statusList = statusList;
+  isLoading: boolean = true;
+  @Input() diff: Diff;
   @Input() editable;
-  @Output() onAddToAttentionList = new EventEmitter<string>();
 
-  // Save needAttentionOf list
-  saveAttentionList(name: string): void {
-    this.onAddToAttentionList.emit(name);
+  // Ask review.component to send diff to firebase
```

Comment:
What's review.component?
I see that the diff is saved in `this.reviewService.saveLocalDiff()`